packages/frontend/src/pages/Send/useSendData.tsx

```diff
@@ -1,18 +1,20 @@
 import { useState, useMemo, useCallback } from 'react'
 import { BigNumber } from 'ethers'
-import { Token } from '@hop-protocol/sdk'
+import { HopBridge, Token } from '@hop-protocol/sdk'
 import { useApp } from 'src/contexts/AppContext'
 import Network from 'src/models/Network'
 import useDebounceAsync from 'src/hooks/useDebounceAsync'
+import useAvailableLiquidity from './useAvailableLiquidity'
 
 const useSendData = (
   token: Token | undefined,
   slippageTolerance: number,
   fromNetwork: Network | undefined,
   toNetwork: Network | undefined,
-  fromAmount: BigNumber | undefined
+  fromAmount: BigNumber | undefined,
+  selectedBridge?: HopBridge
```

Comment:
this can be removed, as we can call `sdk.bridge(token.symbol)` for better readability and consistency and less overhead